client/src/components/new-request/search.vue

```diff
@@ -259,19 +266,13 @@ export default class NewSearch extends Vue {
   }
   @Watch('request_action_cd')
   watchRequestActionCd (newVal) {
-    if (Object.keys(bcMapping).includes(newVal) && ['BC'].includes(this.location)) {
-      let { value } = newReqModule.entityTypesBCData.find(ent => ent.rank === 1)
-      newReqModule.mutateEntityType(value)
-    }
-    if (Object.keys(xproMapping).includes(newVal) && ['CA', 'IN'].includes(this.location)) {
-      let { value } = newReqModule.entityTypesXPROData.find(ent => ent.rank === 1)
-      newReqModule.mutateEntityType(value)
-    }
     // Set default location to BC for the requests where BC is the only location option
     if (['CNV', 'MVE'].includes(newVal)) {
       this.location = 'BC'
+      this.jurisdictionDisabled = true
       return
```

Comment:
Is the jurisdiction field visible under either of these two requests? 
From what i recall, these were BC only, so the jurisdiction field might not even be viewable. 
Nothing major, but it might not be required to disable it if it's not rendered, not sure 🤔

Reply:
hey sorry cam, this is for disabling the 'location'. UX had me change the label to 'select a jurisdiction' so I thought this was a good name, but I could change this to disableLocation to make it less confusing if you like?

Reply:
ahhh, sorry i missed that :) I got confused because we have a dedicated jurisdiction field as well.
Hmm yea, it might be good to have a distinction between the two fields, but not major at all.